plugins/nodebb-plugin-caiz/languages/ja/caiz.json

```
@@ -150,5 +150,84 @@
+  "notification-settings": "通知設定",
+  "slack-notifications": "Slack通知",
+  "enable-slack-notifications": "Slack通知を有効にする",
+  "connect-to-slack": "Slackに接続",
+  "discord-notifications": "Discord通知",
+  "enable-discord-notifications": "Discord通知を有効にする",
+  "connect-to-discord": "Discordに接続",
+  "notification-channel": "通知先チャンネル",
+  "select-channel": "チャンネルを選択",
+  "notification-events": "通知イベント",
+  "notify-new-topic": "新規トピック作成",
+  "notify-new-post": "新規投稿",
```

Biome (2.1.2)

[error] The key categories was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)